packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -520,6 +572,25 @@ export default class VTable {
         this.normalizeTableCellSize();
         setHTMLElementSizeInPx(this.table); // Make sure table width/height is fixed to avoid shifting effect
     }
+
+    /**
+     * Check if the cell is inside of the selection range
+     * @returns true if it is inside, otherwise false
+     */
+    private isInsideOfSelection(x: number, y: number) {

```

Comment:
suggest inline the logic into getSelectedRanges. and some of them only need to do once.